src/supervisor-api.ts

```diff
@@ -87,6 +90,14 @@ export class SupervisorAPI {
 
 	private api = express();
 	private server: Server | null = null;
+	// Holds the function which should apply iptables rules
+	private applyRules: SupervisorAPI['applyListeningRules'] =
+		process.env.TEST === '1'
+			? () => {
+					// don't try to alter iptables
```

Comment:
NIT: The indentation is super weird here, can you make it indented from the opening brace by one please please? I'm not sure why prettier didn't change this

Reply:
Will add this change before merging.